### ISSUE TITLE
youtube

### ISSUE THREAD

**Issue by edsu** [webrecorder/webrecorderplayer-electron#19]
I realize there are a lot of things going on with this, and it's early days for the player, but I noticed that video from YouTube that I recorded with Webrecorder isn't playing in the WebrecorderPlayer but is in WebarchivePlayer:

![screen shot 2017-03-31 at 4 08 10 pm](https://cloud.githubusercontent.com/assets/33829/24567896/99c9c32a-162d-11e7-86a6-176862b6ef6f.png)

![screen shot 2017-03-31 at 4 14 27 pm](https://cloud.githubusercontent.com/assets/33829/24567890/9631839c-162d-11e7-9641-0d79b42f57c8.png)

Maybe it has something to do with Chromium/Electron? I'm using WebarchivePlayer in Firefox, which is what I recorded with in Webrecorder.

**Comment by ikreymer** [comment]
I bet it has something to do with Brotli compression, which should be working by default, but maybe isn't in the player.
Can you double check to see if the page has a `Content-Encoding: br` in the WARC?

**Comment by edsu** [comment]
Yep!

```
WARC/1.0
WARC-Date: 2017-03-31T20:02:10Z
Content-Type: application/http; msgtype=response
WARC-Record-ID: <urn:uuid:eceea858-164c-11e7-9093-0242ac120002>
WARC-IP-Address: 172.217.5.238
WARC-Payload-Digest: sha1:7Q7M5I7DC5JYORDLPQXUHK7DMR2XOX33
WARC-Block-Digest: sha1:UFGG33COEBOCCHSQYO47IIDXBR3OYL6N
Content-Length: 38598

HTTP/1.1 200 OK
X-XSS-Protection: 1; mode=block; report=https://www.google.com/appserve/security-bugs/log/youtube
Content-Encoding: br
Strict-Transport-Security: max-age=31536000
P3P: CP="This is not a P3P policy! See http://support.google.com/accounts/answer/151657?hl=en for more info."
X-Frame-Options: SAMEORIGIN
Content-Type: text/html; charset=utf-8
Cache-Control: no-cache
Expires: Tue, 27 Apr 1971 19:44:06 EST
X-Content-Type-Options: nosniff
Date: Fri, 31 Mar 2017 20:02:10 GMT
Server: YouTubeFrontEnd
Transfer-Encoding: chunked
Alt-Svc: quic=":443"; ma=2592000; v="37,36,35"
Connection: close

<8D>%<F8><91><B2>p<FE>~<F3>ߦ<9E><AE><99>        <ED><E7>#       <B5>
<85>'<E2><DD>N[V<FF>|^<D5><D6>^H\<8A>q<8E><BD><A9>vg<8E><E3>fY䀡=<80><89><82>'6
<F8><A7>)J<F7><86>,<A8><A0>{_<92><FD>j<C9>w<EF><D9>w>_<96>3֐<9D><B1>6<EB>؃<E5>
```

**Comment by ikreymer** [comment]
Released 1.0.1 to fix this issue! https://github.com/webrecorder/webrecorderplayer-electron/releases/tag/v1.0.1

**Comment by edsu** [comment]
Works, thanks!
